src/main/scala/Tile.scala

```diff
@@ -0,0 +1,18 @@
+case class Tile(c: Char) {
+
+  def value: Int = {
+    c match {
+      case 'A' | 'E' | 'I' | 'O' | 'U' | 'N' | 'R' | 'T' | 'L' | 'S' => 1
+      case 'G' | 'D' => 2
+      case 'B' | 'C' | 'M' | 'P' => 3
+      case 'F' | 'H' | 'V' | 'W' | 'Y' => 4
+      case 'K' => 5
+      case 'J' | 'X' => 8
+      case 'Q' | 'Z' => 10
+      case _ => 0
```

Comment:
this case is only needed to keep the compiler happy, but actually we know that this case would be a program bug.  You could use a type that only had valid letters, you'd have a hassle to model it ( a sealed trait or sealed class with 26 different subclasses) but once you had modelled it, the type checker would be able to help out more in the logic parts.
That is part of the classic trade off of the more you help the compiler out, the more it helps you, however you can end up making a lot of code just to put the types in in the first place.